### PR TITLE
[NCP-3457] support propertyHostname in filter and groupBy

### PR DESCRIPTION
**Pull request by jungho73**

### CODE REVIEW

File: en_US.json [hunk 1]
```diff
@@ -2044,6 +2046,8 @@
 	"reportQuery.page.title.type.reqSummary": "Edge Request Summary",
 	"reportQuery.page.title.type.statusDetails": "Status Code Details Chart",
 	"reportQuery.page.title.type.volSummary": "Edge Traffic Volume Summary Chart",
+	"reportQuery.page.validation.count.group.by": "Please select not more than 2 items.",
+	"reportQuery.page.validation.count.wildcard": "Multiple wildcard hostnames is not allowed.",
```
Comment (burtonquantil) [inline en_US.json:2050]: Change the English to:

"Please select no more than two items."
"Multiple wildcard hostnames are not allowed."

**Review by burtonquantil — CHANGES_REQUESTED** [review]
Refer to the suggestions.

File: en_US.json (hunk 1 above)
Comment (jungho73) [reply en_US.json]: I changed some text by the suggestions
Thanks.

File: en_US.json [hunk 2]
```diff
@@ -1921,6 +1921,7 @@
 	"report.vol.label.edge.traffic": "Edge Traffic",
 	"report.vol.label.origin.traffic": "Origin Traffic",
 	"report.vol.label.traffic.vol": "Traffic Volume",
+	"reportQuery.page.check.query.propertyHostnames": "query as property hostname",
```
Comment (burtonquantil) [inline en_US.json:1924]: @jungho73  can you clarify how this text is used?  It sounds a bit strange

Comment (jungho73) [reply en_US.json:1924]: @burtonquantil  Used as a checkbox title below the hostname edit box

Comment (burtonquantil) [reply en_US.json:1924]: @jungho73  OK, I saw Bin's recommendation of this text.  I guess I'll try it when visible in our test environment and will comment if I think it should be reworded. Thanks.

**Review by burtonquantil — APPROVED** [review]
The English version looks OK at this time.

File: zh-Hans.json [hunk 3]
```diff
@@ -1921,6 +1921,7 @@
 	"report.vol.label.edge.traffic": "边缘节点流量",
 	"report.vol.label.origin.traffic": "回源流量",
 	"report.vol.label.traffic.vol": "数据流量",
+	"reportQuery.page.check.query.propertyHostnames": "查询作为属性主机名",
```
Comment (Jianghai-Quantil) [inline zh-Hans.json:1924]: Please change it to "按加速域名查询"

File: zh-Hans.json [hunk 4]
```diff
@@ -1935,6 +1936,7 @@
 	"reportQuery.page.group.by.hostAndServer": "域名 + 服务群",
 	"reportQuery.page.group.by.hostnames": "域名",
 	"reportQuery.page.group.by.none": "无",
+	"reportQuery.page.group.by.propertyNames": "属性主机名",
```
Comment (Jianghai-Quantil) [inline zh-Hans.json:1939]: Please change it to "加速域名"

File: zh-Hans.json [hunk 5]
```diff
@@ -2044,6 +2046,8 @@
 	"reportQuery.page.title.type.reqSummary": "边缘请求数汇总",
 	"reportQuery.page.title.type.statusDetails": "状态码详情图表",
 	"reportQuery.page.title.type.volSummary": "边缘流量汇总图",
+	"reportQuery.page.validation.count.group.by": "请选择不超过两项。",
```
Comment (Jianghai-Quantil) [inline zh-Hans.json:2049]: Please change it to "最多只能选择两项。"

File: zh-Hans.json [hunk 6]
```diff
@@ -2044,6 +2046,8 @@
...
+	"reportQuery.page.validation.count.wildcard": "不允许使用多个通配符主机名。",
```
Comment (Jianghai-Quantil) [inline zh-Hans.json:2050]: Please change it to "不按加速域名查询时，最多只能选择一个泛域名。"

**Review by Jianghai-Quantil — CHANGES_REQUESTED** [review]
Requested some changes to the Chinese version

File: zh-Hans.json (hunk 4 above)
Comment (jungho73) [reply zh-Hans.json]: changed this.

File: zh-Hans.json (hunk 5 above)
Comment (jungho73) [reply zh-Hans.json]: changed this.

File: zh-Hans.json (hunk 6 above)
Comment (jungho73) [reply zh-Hans.json]: changed this.

**Comment by jungho73** [comment]
> Requested some changes to the Chinese version
I've modified some of the text as suggested.

**Review by Jianghai-Quantil — APPROVED** [review]
The changes look good to me now.

**Pull request merged**